eng/testing/performance/performance-setup.ps1

```diff
@@ -28,6 +28,41 @@ Param(
     [switch] $iOSLlvmBuild
 )
 
+function Verified-Move-Item {
+	[Parameter(Mandatory)]
+	[string]$Path
+	[Parameter(Mandatory)]
+	[string]$Destination
+	
+	Move-Item -Path $Path -Destination $Destination
+	if (!$?) {
+		Write-Output "Failed to move $Source to $Destination"
+		exit 1
+	}
+}
+
+function Verified-Copy-Item {
+	[Parameter(Mandatory)]
+	[string]$path
+	[Parameter(Mandatory)]
+	[string]$Destination
+	
+	Copy-Item -path $path $Destination
+	if (!$?) {
+		Write-Output "Failed to copy $Source to $Destination"
+		exit 1
+	}
+}
+
+function Verify-Robocopy {
+	[Parameter(Mandatory)]
+	[string]$Source
+	if ($LASTEXITCODE -ne 0 -or !$?) {
+		Write-Output "Failed to copy $Source: exit code $LASTEXITCODE"
+		exit $LASTEXITCODE
+	}
+}
+
```

Comment:
FWIW this isn't really the pattern we were looking for but I suppose it's fine. for example, if you had written a generic function that took a string, you could have used it for `git` and `robocopy` invocations. You also wouldn't have to wrap the cmdlets being called.